webservices/docs.py

```diff
@@ -339,11 +340,10 @@
 
 Of those transactions,[under $200, and having "earmark" in the memo text OR transactions having the codes 11A, 12, 17, 17A, or 18], we then want to exclude earmarks.
 
-This is [the sql function](https://github.com/fecgov/openFEC/blob/develop/data/functions/individual.sql) that defines individual contributions:
 '''
 
 SCHEDULE_A = SCHEDULE_A_TAG + '''
-The data is divided in two-year periods, called `two_year_transaction_period`, which
+Individual contributions data is divided in two-year periods, called `two_year_transaction_period`, which
```

Comment:
@patphongs sorry I missed this before - Schedule A isn't just for individual contributions - it's all "receipts"